### PR TITLE
v2.0.0

### PR DESCRIPTION
**Pull request by Ar2emis**
- Refactored
- Added pretty cases, handler,  endpoint_options and before_response definitions

### CODE REVIEW

**Review by differencialx — CHANGES_REQUESTED** [review]

File: README.md [hunk 1]
```diff
@@ -91,6 +117,54 @@ end
...
+  def is_it_raining?
+    WeatherForecast.for_today.raining?
+  end
+
+  def is_vasya_in_the_house?
+    User.find_by(login: 'vasya').signed_in?
+  end
+end
+```
+If you want to remove parent cases use `inherit` option
```
[the remark this answers is not in the document]
Comment (Ar2emis) [reply README.md:143]: Description isn't in code block. It's between two code blocks, isn't it?

File: lib/simple_endpoint/errors.rb [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    Please implement default cases conditions via hash
+
+    EXAMPLE:
+    ###############################################
+    # default trailblazer-endpoint logic, you can change it
+    # Can be put into ApplicationController and redefined in subclasses
+
+    private
+
+    def default_cases
```
[the remark this answers is not in the document]
Comment (Ar2emis) [reply lib/simple_endpoint/errors.rb:56]: done

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (differencialx) [reply README.md:139]: Yeah, sorry )

**Review by differencialx — APPROVED** [review]

**Comment by differencialx** [comment]
@Ar2emis Thank you for such large contribution. Great work!!! :+1:

**Pull request merged**